client/ngapp/scripts/controllers/country.js

```diff
@@ -24,6 +23,15 @@ angular.module('ooniAPIApp')
         console.log('error', error)
       })
 
+    Country.findOne({
```

Comment:
Oh hey, actual change above is here, the above comment was on the compiled file. (Should we remove the compiled file form the repo by the way?)